.travis.yml

```diff
@@ -60,14 +60,15 @@ install:
       fi;
     - |
       if [ "$DISTRIB" == "ubuntu" ]; then
-        # Use a separate virtual environment that the one provided by
+        # Use a separate virtual environment than the one provided by
         # Travis because it contains numpy and we want to use numpy
         # from apt-get
         deactivate
         virtualenv --system-site-packages testvenv
         source testvenv/bin/activate
+        pip install -U requests[security]  # ensure SSL certificate works
         pip install -r requirements.txt
-        pip install seaborn sphinx==1.4.9 pytest "six>=1.10.0" pytest-cov
+        pip install seaborn sphinx==1.5.5 pytest pytest-cov
```

Comment:
The git merge from master is working against us

Reply:
I was crossing my fingers for this but it was not enough apparently ... I rebased on master in the mean time.